docs/contributing.md

```diff
@@ -185,20 +188,20 @@ the browser will automatically show the new content.
 
 Create a `puml` file inside `docs/diagrams/src`
 
-```
+```sh
 touch docs/diagrams/src/demo.puml
 ```
 
 Check the PlantUML syntax here: <https://plantuml.com/deployment-diagram>
 
 Serve docs site on localhost:
 
-```
+```sh
 mkdocs serve
 ```
 
 Add the yielded `svg` into the desired `.md` file
 
-```
-![Diagram demo](diagrams/out/demo.svg "Diagram demo")
-```
+```markdown
```

Comment:
TIL
it rendered nicely: https://logserver.rdoproject.org/92/492/b087544d1760fa13d2754b4ddde0978a17411e46/github-check/dataplane-operator-docs-preview/6b8696d/docs/contributing/